mikeio/res1d.py

```diff
@@ -1,141 +1,274 @@
+from collections import defaultdict, namedtuple
+from contextlib import contextmanager
+
 import clr
 import os.path
 import pandas as pd
 
 clr.AddReference("DHI.Mike1D.ResultDataAccess")
-from DHI.Mike1D.ResultDataAccess import ResultData
+from DHI.Mike1D.ResultDataAccess import ResultData  # noqa
 
 clr.AddReference("DHI.Mike1D.Generic")
-from DHI.Mike1D.Generic import Connection
+from DHI.Mike1D.Generic import Connection  # noqa
```

Comment:
# noqa. What does this mean?

Reply:
That's how to tell Python linters to skip that line, see https://stackoverflow.com/questions/45346575/what-does-noqa-mean-in-python-comments

I've added that comment here since my linter in VSCode was complaining about this import as it couldn't resolve it (due to the fact that this import is resolved at runtime). I assume other linters will have the same error in other editors, if you want to add a linter check in your CI in the future it's likely you'll have to add `# noqa` at the end of each one of these imports (there might be a global setting though).

Reply:
I can remove it if you like!

Reply:
No need. It has a purpose.